### ISSUE TITLE
handle null in default ruleset

### ISSUE THREAD

**Issue by xquery** [marklogic/marklogic-sesame#180]
in MarkLogicClientImpl should add get/set for setting of default rulesets which filter out null.

**Comment by srinathgit** [comment]
I think the same would apply to set/getDefaultQueryDef as well.

**Comment by xquery** [comment]
yes to 1) I disagree for point 2) ... we are mirroring the primitives exposed by java api client.

**Comment by xquery** [comment]
filter null from defaultRuleset

**Comment by xquery** [comment]
filter null from defaultGraphPerms

**Comment by grechaw** [comment]
Issue migrated to jira for marklogic-rdf4j and is in current sprint planning.
